radixdlt/src/main/java/com/radixdlt/consensus/bft/VertexStore.java

```diff
@@ -72,26 +71,6 @@ public VertexStore(
 		BFTUpdateSender bftUpdateSender,
 		VertexStoreEventSender vertexStoreEventSender,
 		SystemCounters counters
-	) {
-		this(
-			rootVertex,
-			rootQC,
-			Collections.emptyList(),
-			ledger,
-			bftUpdateSender,
-			vertexStoreEventSender,
-			counters
-		);
-	}
-
-	public VertexStore(
-		VerifiedVertex rootVertex,
-		QuorumCertificate rootQC,
-		List<VerifiedVertex> vertices,
-		Ledger ledger,
-		BFTUpdateSender bftUpdateSender,
-		VertexStoreEventSender vertexStoreEventSender,
-		SystemCounters counters
```

Comment:
Will need the constructor with vertices once we start implementing recovery as well as for better unit tests. Would rather not remove these for the sake of cleanliness as then we lose the information that this original intended constructor works.

Reply:
OK, I've put it back for now.